OpenRA.Mods.RA/Activities/Infiltrate.cs

```diff
@@ -56,5 +58,13 @@ protected override void OnInside(Actor self)
 				Game.Sound.PlayNotification(self.World.Map.Rules, self.Owner, "Speech",
 					notification, self.Owner.Faction.InternalName);
 		}
+
+		public override Activity Tick(Actor self)
+		{
+		if (self.Trait<Infiltrates>().IsTraitDisabled)
```

Comment:
I never thought about multipile Infiltrates: traits. I was just trying to enable/disable it. So how and where the check should be?